Picturesbooks.json

```diff
@@ -0,0 +1,51 @@
+[
```

Comment:
What file is this?
This was added in previous PR too

Reply:
that file also not require.

Reply:
Remove the file and push the changes to your branch